docs/source/releases/latest/final-CLI-updates-before-workshop.yaml

```diff
@@ -0,0 +1,25 @@
+Regression Fix:
```

Comment:
Do not merge until https://github.com/NRLMMD-GEOIPS/geoips/pull/724 is merged.  These CLI updates will go away when #724 is merged.